services/order.py

```diff
@@ -0,0 +1,27 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> Order:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user, created_at=date)
+
+    for ticket in tickets:
+        Ticket.objects.create(
+            movie_session_id=ticket["movie_session"],
```

Comment:
Ensure that the `movie_session` key exists in the `ticket` dictionary to avoid potential `KeyError`. Consider adding validation or error handling for the input data.